core/app_modules/inventory.js

```diff
@@ -103,6 +102,9 @@ inventory.check_out = co.wrap(function*(items, patron, employee) {
 
       var q = 'INSERT INTO checked_out(call, copy, netid, attendant, due)' +
         'VALUES($1, $2, $3, $4, $5)';
+        
+      var notes ='Item checked out by ' + employee + ' due on ' + due.toString().substring(0,10);
```

Comment:
The substring excludes a year (and may have unexpected results due to variances in server locales). Is there room to store the whole date (i.e., due.toString())?